api-reference/10 UI Components/dxForm/1 Configuration/isDirty.md

```diff
@@ -5,7 +5,12 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether editors current values differ from the initial values.
```

Comment:
"editors current values" reads wrong to me. First of all, "editor" needs to act as an adjective for the word "value". And it doesn't matter if you talk about a value (singular) or values (plural). The modifier (adjective) stays the same - "editor". You can say "specifies whether the editor value" or "specifies whether editor values". 

Now the word "current" should precede this noun phrase: "current editor values". Otherwise you'd have to switch to possessive - "editors' current values". And it's best to avoid plural possessives. 

Finally, you don't need an article before "initial values". It's plural and not really a specific reference. 

```suggestion
Specifies whether current editor values differ from initial values.
```